index.js

```diff
@@ -0,0 +1,55 @@
+
+const selectMenu = document.querySelector("select")
+
+
+const getGhibliMovies = async () => {
+    try{
+        const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+        const moviesArr = res.data;
+        moviesArr.forEach(movie => {
+            const option = document.createElement("option");
+            option.textContent = movie.title
+            option.value = movie.id 
+            selectMenu.appendChild(option);
+        });
+        
+    } catch (err) {
+        console.log(err)
+    }
+}
+
+getGhibliMovies();
+
+const title = document.createElement("h3");
+const displayInfo = document.querySelector("#display-info");
+
+selectMenu.addEventListener("click", async (e) => {
+    try {
+        displayInfo.textContent = ""
+        const url = `https://ghibliapi.herokuapp.com/films/${e.target.value}`;
+        const res = await axios.get(url);
+        const relYear = document.createElement("p");
+        const description = document.createElement("p");
+        title.textContent = res.data.title;
+        relYear.textContent = res.data.release_date;
+        description.textContent = res.data.description;
+        displayInfo.appendChild(title);
+        displayInfo.appendChild(relYear);
+        displayInfo.appendChild(description);
+    } catch (err) {
+        console.log(err);
+    }
```

Comment:
We can define the `title, releaseYear, and description`  at the top of the page and just replace the `.innerText` when ever we get new data from the api call